script/mimic

```diff
@@ -0,0 +1 @@
+Subproject commit 203f85db5de57703e7100a00636060f3095dda29
```

Comment:
In order to run it inside Docker, I checked out the mimic repo and have it living here. Does git magically make that a submodule?